src/Template/Bake/Element/form.ctp

```diff
@@ -71,9 +71,17 @@ $fields = collection($fields)
                 continue;
             }
             if (!in_array($field, ['created', 'modified', 'updated'])) {
+                $fieldData = $schema->column($field);
+                if (($fieldData['type'] === 'date') && (!empty($fieldData['null']))) {
+%>
+            echo $this->Form->input('<%= $field %>', array('empty' => true, 'default' => ''));
+<%
+                } else {
 %>
             echo $this->Form->input('<%= $field %>');
 <%
+                }
+                continue;
```

Comment:
Oops !

I don't know what just happened.  I refreshed the screen, now the quoted code is the same as mine.